app/finders/conversation_finder.rb

```diff
@@ -155,7 +155,7 @@ def current_page
   end
 
   def conversations
-    @conversations = @conversations.includes(:taggings, { inbox: [:channel] },
```

Comment:
Had to remove this, was getting an error in the tests (See previous commit). Will try to fix this.

```
ActiveRecord::EagerLoadPolymorphicError Exception: Cannot eagerly load the polymorphic association :channel
```

CC: @sojan-official